pkg/storegateway/blocks.gohtml

```diff
@@ -90,5 +103,92 @@
     {{ end }}
     </tbody>
 </table>
+<script>
```

Comment:
Maybe we could use something like htmlx.org?

Reply:
for the simple submit i replace with htmx function, for the post, i need to parse the table and get the list of ulid to send, for that I kept the scripts.